bin/spark-submit2.cmd

```diff
@@ -1,78 +0,0 @@
-@echo off
-
```

Comment:
Hmm. I think I messed up some intermediate diff. This should be calling `spark-class`.